git.d

```diff
@@ -44,28 +59,13 @@ private:
 public // So std.parallelism can get at it
 StatusFlags asyncGetFlags(Duration allottedTime)
 {
-	// Currently we can only do this for Unix.
-	// Windows async pipe I/O (they call it "overlapped" I/O)
-	// is more... involved.
-	// TODO: Either write a Windows implementation or suck it up
-	//       and do things synchronously in Windows.
-	import core.sys.posix.poll;
-
 	StatusFlags ret;
 
-	// Light off git status while we find the HEAD
-	auto pipes = pipeProcess(["git", "status", "--porcelain"], Redirect.stdout);
```

Comment:
The process was intentionally started here so it can run while we do other work below.